demo_exam/Functions.h

```diff
@@ -0,0 +1,95 @@
+#pragma once
+const double EPSILON = 0.01;

```

Comment:
Normally we want epsilon to be a very small number, so something like 0.000001 would be better (if it's not stated otherwise in the task itself).